Google.Api.Generator.Rest.Tests/GoldenTestData/Google.Apis.Assuredworkloads.v1/Google.Apis.Assuredworkloads.v1.cs

```diff
@@ -644,13 +644,88 @@ public ListRequest(Google.Apis.Services.IClientService service, string parent) :
                         [Google.Apis.Util.RequestParameterAttribute("filter", Google.Apis.Util.RequestParameterType.Query)]
                         public virtual string Filter { get; set; }
 
+                        private string _intervalEndTimeRaw;
+
+                        private object _intervalEndTime;
+
                         /// <summary>The end of the time window.</summary>
                         [Google.Apis.Util.RequestParameterAttribute("interval.endTime", Google.Apis.Util.RequestParameterType.Query)]
-                        public virtual object IntervalEndTime { get; set; }
+                        public virtual string IntervalEndTimeRaw
```

Comment:
Read first my comment on the doc, but:

I'm not certain what effect will this have on the request "serialization", before it was serializing an object, and now it's serializing a string. I think you need the raw property to control how the value is serialized for the request, but I don't think it needs to be publicly settable. I think users should only set the DTO property, and we set the Raw property based on that with the format we want to serialize the value, and keep the `RequestParameterAttribute` on the raw one so that no more serialization happens.

Reply:
Will make all comments on the doc.